pandas/errors/__init__.py

```diff
@@ -65,3 +65,7 @@ class MergeError(ValueError):
     Error raised when problems arise during merging due to problems
     with input data. Subclass of `ValueError`.
     """
+
+
+class AccessorRegistrationWarning(Warning):
```

Comment:
do we really need a special warning here?

Reply:
Switched to a UserWarning, which matches what we do with `df.x = [1, 2, 3]`.